### ISSUE TITLE
Follow up question/bug report regarding #3390

### ISSUE THREAD

**Issue by Folas1337** [GC-spigot/AdvancedEnchantments#3404]
### Describe the bug

I want to reopen the following issue or at least get a definitive answer to the issue: https://github.com/GC-spigot/AdvancedEnchantments/issues/3390

This issue isn't really fixed, I tried it with AE 9.2.0 and now the Terraform ability does nothing when you activate it.

It's supposed to break a lot of extra blocks but now it doesn't do that, I am however afraid that they are somewhat mutually exclusive as Terraform really breaks a heap of blocks in the same Y axis and Trench is attempting to break extra blocks for each one that was broken from Terraform.

I'm not really sure if there is a proper way to deal with this, I just want to express that I dislike how it's currently handled and that it's kind of a downgrade from what it used to be.

### How to reproduce

(see earlier issue)

### Screenshots / Videos

(see earlier issue)

### "/ae plinfo" link

(see earlier issue)

### Server Log

(see earlier issue)

**Comment by GC-spigot** [comment]
Resolved

**Comment by Folas1337** [comment]
> Resolved

I'm afraid that's not really true. I stated that Trench from AE + the skill Excavation from AureliumSkills used to be really powerful and now it's basically just trench with no extra perks if you use the skill Excavation.

I would like this to be re-opened, I verified it with AE 9.3.6 and AureliumSkills 1.3.21 to not do anything other than trench would do.

Either tell me you won't do it (and that's totally fine by the way, I just want to know for certain) or keep it open until you get around to **actually** fixing it.

Thanks in advance :)
